### ISSUE TITLE
ReferenceError: exports is not defined in ES module scope

### ISSUE THREAD

**Issue by tecoad** [allwqre/react-responsive-font-size#1]
Hi, 

I tried this package on a Node.js project using TS, however get this went rendering:

```code
Server Error
ReferenceError: exports is not defined in ES module scope
This file is being treated as an ES module because it has a '.js' file extension and '***/node_modules/react-responsive-font-size/package.json' contains "type": "module". To treat it as a CommonJS script, rename it to use the '.cjs' file extension.
```

Know how to fix it?

BR

**Comment by allwqre** [comment]
Hi, yes.
This seems to be an issue with the package type I've set in the package.json.

I've just published an updated version (1.1.3) which should fix it.
Please update the package, try again and let me know whether it fixed the problem.

**Comment by allwqre** [comment]
Closing this due to lack of feedback.
